charts/images.yaml

```
@@ -28,6 +28,12 @@ images:
   repository: k8scloudprovider/openstack-cloud-controller-manager
   tag: "v1.20.1"
   targetVersion: ">= 1.20"
```

You have to adapt this version to be `targetVersion: "1.20.x"`. Otherwise it will pick randomly as both predicates will match.

```suggestion
  targetVersion: "1.20.x"
```